src/question-two/QuestionTwo.tsx

```diff
@@ -8,7 +8,6 @@ interface ResourceSchedule {
   resourceName: string
   resourceId: number
   allocations: {
-    allocId: number,
```

Comment:
Honestly, didn't think we need this. They need to hit the `allocId` anyway to merge stuff. Safe to scrap.